scripts/workers/rl_submission_worker.py

```diff
@@ -247,6 +250,27 @@ def update_failed_jobs_and_send_logs(
                         )
 
 
+def install_gpu_drivers(api_instance):
+    """Function to get the status of a running job on AWS EKS cluster
+    Arguments:
+        api_instance {[AWS EKS API object]} -- API object for creating deamonset
+    """
+    logging.info("Install GPU Drivers.")
+    link = "https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.11/nvidia-device-plugin.yml"  # pylint: disable=line-too-long
+    logging.info("Using daemonset file: %s", link)
+    f = urllib.urlopen(link)
```

Comment:
Can you please change `f` to something meaningful?

Reply:
Sure